ggml-sycl.cpp

```diff
@@ -13451,44 +13441,35 @@ static void ggml_sycl_mul_mat_mat_batched_sycl(const ggml_tensor *src0,
                                          {sycl::aspect::fp16});
 
             main_stream->submit([&](sycl::handler &cgh) {
-                const sycl::half *src1_as_f16_get_ct1 = src1_as_f16.get();
-                const void **ptrs_src_get_ct3 = ptrs_src.get();
-                void **ptrs_dst_get_ct4 = ptrs_dst.get();
-
+                const void **ptrs_src_get = ptrs_src.get();
+                void **ptrs_dst_get = ptrs_dst.get();
+                auto nb12_scaled = src1->type == GGML_TYPE_F16 ? nb12 : nb12 / 2;
```

Comment:
```suggestion
                sycl::half nb12_scaled = src1->type == GGML_TYPE_F16 ? nb12 : nb12 / 2;
                sycl::half nb13_scaled = src1->type == GGML_TYPE_F16 ? nb12 : nb13 / 2;
```

Reply:
I have set the `scaled` variables to `size_t` as that is the type that `k_compute_batched_ptrs` accepts.